src/analyses/README.md

```diff
@@ -92,7 +92,64 @@ To be documented.
 
 \subsection analyses-dependence-graph Data- and control-dependence analysis (dependence_grapht)
 
-To be documented.
+### Dependence graph
+
+Implemented in `src/analyses/dependence_graph.h(cpp)`. It is a graph and an
+abstract interpreter in the same time. The abstract interpretation nature
+allows a dependence graph to build itself (the graph) from a given GOTO program.
+
+A dependence graph extends the class `grapht` with `dep_nodet` as the type of
+nodes (see `src/util/graph.h` for more details about
+[graph representation](../util/README.md)).
+The `dep_nodet` extends `graph_nodet<dep_edget>` by an iterator to a GOTO
+program instruction. It means that each graph node corresponds to a particular
```

Comment:
what is "It" here?

Reply:
The information of the previous statement - the fact that each node stores an iterator to a GOTO instruction.